integration-tests/api/__tests__/store/__snapshots__/products.js.snap

```diff
@@ -1,450 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
question: why is this deleted?

Reply:
Because it has been replaced by the expect instead. As we agreed some times ago that the snap will disapeared

Reply:
Sorry wasn't aware :)

Reply:
ahah, I think it started here https://medusacorp.slack.com/archives/C025KMS13SA/p1668178263149179